dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/fat/tests/implicit/ImplicitWarLibJarsTest.java

```diff
@@ -88,10 +90,10 @@ public static void buildShrinkWrap() throws Exception {
                         .addAsLibrary(implicitBeanExplicitArchive)
                         .addAsLibrary(utilLib);
 
+       server.setMarkToEndOfLog(server.getDefaultLogFile());
        ShrinkHelper.exportDropinAppToServer(server, implicitEJBInWar);
        ShrinkHelper.exportDropinAppToServer(server, implicitBeanDiscovery);
-
-       server.waitForStringInLogUsingMark("CWWKZ0001I.*Application implicitBeanDiscovery started");
+       assertNotNull("implicitBeanDiscovery started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*implicitBeanDiscovery"));
```

Comment:
will implicitBeanDiscovery always start after implicitEJBInWar? Do you actually need to check both?

Reply:
Yeah, probably should check both